main.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	"net/http"
+)
+
+func helloHandler(w http.ResponseWriter, r *http.Request) {
+	w.Write([]byte("Hello world!"))
+}
+
+func main() {
+	http.HandleFunc("/hello", helloHandler)
```

Comment:
Any particular reason not to serve at the root URL (`/`)?

Reply:
I think `/hello` makes it a little more clear that this is a endpoint mapping compared to just a `/`

Reply:
Great point—I didn't think of that.  Yay for being as specific as possible!

Reply:
I agree with @merenbach  -- we should just be able to visit the URL (without going to /hello) and see the webpage. Otherwise we will hit a 404, which doesn't make for a good demo.